addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/ContentSecurityPolicyScanRule.java

```diff
@@ -435,6 +476,31 @@ private void raiseAlert(
                 .raise();
     }
 
+    private AlertBuilder buildAlert(
```

Comment:
`raiseAlert` should make use of this method.

Reply:
Since it's now passing the refs, it can be changed to a "raiseAlert" (they all call raise without changing the builder now).

Reply:
k :grinning: